### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by eledwinn**
## Summary
- Add responsive design to index

## Screenshot
dec-19-2018 15-41-07

## Trello Card
https://trello.com/c/09VPBERC/15-dise%C3%B1o-responsive

### CODE REVIEW

File: scss/footer.scss [hunk 1]
```diff
@@ -17,3 +17,18 @@
 .footer-copyright, .footer-note {
   color: $white;
 }
+
+.footer-image {
+  max-width: 90%;
```
Comment (FrankIglesias) [inline scss/footer.scss]: why this width?

File: scss/footer.scss [hunk 2]
```diff
@@ -17,3 +17,18 @@
...
+}
+
+@media only screen and (max-width: 1024px) {
+  .footer-image {
+    margin-bottom: 30px;
+  }
+  
+  .footer-copyright {
+    align-self: inherit;
+    margin: 50px 0px;
```
Comment (FrankIglesias) [inline scss/footer.scss]: remove the px suffix

File: scss/styles.scss [hunk 3]
```diff
@@ -31,3 +33,9 @@
 .p-top-nav {
   padding-top: $navbar-height;
 }
+
+@media only screen and (max-width: 1024px) {
+  .body{
```
Comment (FrankIglesias) [inline scss/styles.scss]: add a space before the curly brace

File: scss/titles.scss [hunk 4]
```diff
@@ -14,3 +14,10 @@
     z-index: -1;
   }
 }
+
+@media only screen and (max-width: 1024px) {
+  .tech-title {
+    margin-top: 110px;
+  }
+
```
Comment (FrankIglesias) [inline scss/titles.scss]: remove this line

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: scss/footer.scss (hunk 1 above)
Comment (eledwinn) [reply scss/footer.scss]: When screen width is less than image width, the screen looks like this:

image

That's why the with property was added.

Comment (FrankIglesias) [reply scss/footer.scss]:
```scss
width: 100%;
max-width: 300px;
```

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**